site/en/r2/tutorials/generative/dcgan.ipynb

```diff
@@ -1,143 +1,197 @@
 {
```

Comment:
<p>Maybe add a comment like: </p><p>"The model output is a binary-logit. It will be trained to output positive values for real images and negative values for fake images. The current output is near zero, indicating uncertainty, which is what you expect for an untrained model.  </p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Cool, done.</p>